interfaces/netinfo.idl

```diff
@@ -4,42 +4,39 @@
 // See: https://wicg.github.io/netinfo/
 
 enum ConnectionType {
-    "bluetooth",
-    "cellular",
-    "ethernet",
-    "mixed",
-    "none",
-    "other",
-    "unknown",
-    "wifi",
-    "wimax"
+  "bluetooth",
+  "cellular",
+  "ethernet",
+  "mixed",
+  "none",
+  "other",
+  "unknown",
+  "wifi",
+  "wimax"
 };
+
 enum EffectiveConnectionType {
-    "2g",
-    "3g",
-    "4g",
-    "slow-2g"
+  "2g",
+  "3g",
+  "4g",
+  "slow-2g"
 };
-[NoInterfaceObject,
- Exposed=(Window,Worker)]
+
+[NoInterfaceObject, Exposed=(Window,Worker)]
 interface NavigatorNetworkInformation {
-    readonly attribute NetworkInformation connection;
+  readonly attribute NetworkInformation connection;
 };
 
 Navigator implements NavigatorNetworkInformation;
-
 WorkerNavigator implements NavigatorNetworkInformation;
+
 [Exposed=(Window,Worker)]
 interface NetworkInformation : EventTarget {
-    readonly attribute ConnectionType          type;
-    readonly attribute EffectiveConnectionType effectiveType;
-    readonly attribute Megabit                 downlinkMax;
-    readonly attribute Megabit                 downlink;
-    readonly attribute Millisecond             rtt;
-    readonly attribute boolean                 saveData;
-             attribute EventHandler            onchange;
-};
-
-typedef unrestricted double Megabit;
-
-typedef unsigned long long Millisecond;
+  readonly attribute ConnectionType type;
+  readonly attribute EffectiveConnectionType effectiveType;
+  readonly attribute Megabit downlinkMax;
+  readonly attribute Megabit downlink;
+  readonly attribute Millisecond rtt;
+  readonly attribute boolean saveData;
+  attribute EventHandler onchange;
+};typedef unrestricted double Megabit;typedef unsigned long long Millisecond;
```

Comment:
This matches https://wicg.github.io/netinfo/#-dfn-networkinformation-dfn-interface and https://github.com/tidoust/reffy-reports/blame/master/whatwg/idl/netinfo.idl but seems like a bug, filed https://github.com/WICG/netinfo/issues/74.